x-pack/plugins/observability/public/pages/alerts/styles.scss

```diff
@@ -0,0 +1,12 @@
+$fullscreenFlyoutTop: 72px;
```

Comment:
This "magic number" makes me a little nervous. There are sass variables for things like the `kbnHeaderOffset`: https://github.com/elastic/kibana/blob/master/src/core/public/_variables.scss#L6, but there doesn't seem to be anything suitable for here and this use case.